packages/enzyme-utilities/src/test/trigger.test.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import {trigger} from '..';
+import {Toggle} from './components/Toggle';
+import {ActionList, Action} from './components/Actions';
+
+describe('trigger', () => {
+  it('calls functions on props with arguments', () => {
```

Comment:
I know this is mostly just moved, but this feels super under tested

Reply:
https://github.com/Shopify/quilt/issues/129